ci/install.sh

```diff
@@ -23,7 +23,7 @@ if [[ "$CONDA" == "true" ]]; then
     # Useful for debugging any issues with conda
     conda info -a
 
-	sed -i -e "s/- python=.*/- python=$TRAVIS_PYTHON_VERSION/g" environment.yml
+	sed -i -e "s/- python=.*/- python=$PYTHON_VERSION/g" environment.yml
```

Comment:
I am confused by this change ... looks independent of the Py3.6 -> Py 3.7 step .. was this a bug before?

Reply:
Yes it was. It was revealed in #1472 because it should have error on 3.6 but didn't since it really was using 3.7